serverless/event_sources/serverless-kafka-source.adoc

```diff
@@ -6,7 +6,9 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The Apache Kafka event source brings messages into Knative. It reads events from an Apache Kafka cluster and passes these events to an event sink so that they can be consumed. You can use the `KafkaSource` event source with {ServerlessProductName}.
+The Apache Kafka event source brings messages into Knative. It reads events from an Apache Kafka cluster and passes these events to an event sink so that they can be consumed.
+
+You can use the `KafkaSource` event source with {ServerlessProductName} after you have xref:../../serverless/admin_guide/installing-knative-eventing.adoc#installing-knative-eventing[Knative Eventing] and xref:../../serverless/event_workflows/serverless-kafka.adoc#serverless-kafka[Apache Kafka] installed on your cluster.
```

Comment:
Maybe better would be to say: Knative Kafka instead of Apache Kafka. Our supported configuration is Knative Kafka with AMQ Streams.